tests/core_test/containers_test/mels_repertoire_test.py

```diff
@@ -0,0 +1,230 @@
+import jax.numpy as jnp
+import pytest
+
+from qdax.core.containers.mels_repertoire import MELSRepertoire
+from qdax.types import ExtraScores
+
+
+def test_mels_repertoire() -> None:
```

Comment:
Minor suggestion: we could use a short sentence to define the scope of the test

Reply:
Updated the test names and added docstrings